multi-arch/packages/layers/X/build.yaml

```diff
@@ -12,6 +12,7 @@ prelude:
 - cd /var/db/repos/gentoo/net-dialup/ppp && rm Manifest && ebuild ppp-2.4.8.ebuild digest
 - eval 'mkdir /usr/share/mocaccino/layer -p || true' && qlist -ICv > /usr/share/mocaccino/layer/{{ .Values.name }}.prelude.pkglist
 steps:
+- cp -rvf package.use/* /etc/portage/package.use/
```

Comment:
Why simply add the fix on mocaccino-overlay-x package with all use flags?

Reply:
It the same for me, just depend on the mocaccinoOS philosophy on use flags handling. 

- One big .use file common in mocaccino-overlay-X package
- split .use files across package, with only use flag on provided package

Either way seem's correct to me, it's just a matter of maintenance choice. The big  .use flag may force more revdep rebuilt than needed.

Reply:
yeah, I agree, but for core-libs it's better to have one point of control, IMHO.

And in plus I will use it soon for single packages review :D